src/components/therapy/TreatmentPlanManager.tsx

```
@@ -0,0 +1,651 @@
+            {renderGoalsSection(newPlanData.goals, false)}
+          </div>
+        </form>
+      </Dialog>
+
+      <AlertDialog open={!!planToDelete} onOpenChange={(isOpen) => !isOpen && setPlanToDelete(null)}>
+        <AlertDialogContent>
+          <AlertDialogHeader>
+            <AlertDialogTitle>Are you absolutely sure?</AlertDialogTitle>
+            <AlertDialogDescription>
+              This action cannot be undone. This will permanently delete the treatment plan
+              titled "<strong>{planToDelete?.title}</strong>" and all its associated goals and objectives.
```

ESLint

[error] `"` can be escaped with `&quot;`, `&ldquo;`, `&#34;`, `&rdquo;`.

(react/no-unescaped-entities)